src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -27,11 +27,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        ' Protected MustOverride Function StatementKind() As OperationKind
-
         Protected Overridable Function StatementKind() As OperationKind
```

Comment:
I see. Then I will probably need to make `Accept` methods in `BoundExpression` and `BoundStatement` abstract/MustOverride too.